src/Symfony/Bundle/FrameworkBundle/Templating/TemplateNameParser.php

```diff
@@ -72,4 +72,9 @@ public function parse($name)
 
         return $this->cache[$name] = $template;
     }
+
+    private function isAbsolute($path)
```

Comment:
This won't work with windows paths which are "normalised" for some reasons here: 

```php
$name = str_replace(':/', ':', preg_replace('#/{2,}#', '/', str_replace('\\', '/', $name)));
```